fe2-android/app/src/main/java/com/github/dedis/student20_pop/utility/handler/GenericHandler.java

```diff
@@ -0,0 +1,155 @@
+package com.github.dedis.student20_pop.utility.handler;
+
+import static com.github.dedis.student20_pop.utility.handler.MessageHandler.handleMessage;
+
+import android.util.Log;
+import com.github.dedis.student20_pop.model.Lao;
+import com.github.dedis.student20_pop.model.data.LAORepository;
+import com.github.dedis.student20_pop.model.data.LAOState;
+import com.github.dedis.student20_pop.model.network.GenericMessage;
+import com.github.dedis.student20_pop.model.network.answer.Error;
+import com.github.dedis.student20_pop.model.network.answer.Result;
+import com.github.dedis.student20_pop.model.network.method.Broadcast;
+import com.github.dedis.student20_pop.model.network.method.message.MessageGeneral;
+import io.reactivex.subjects.Subject;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/** Subscribe, catchup, create LAO and broadcast handler class */
+public class GenericHandler {
+
+  public static final String TAG = GenericHandler.class.getSimpleName();
+
+  private GenericHandler() {
+    throw new IllegalStateException("Utility class");
+  }
+
+  /**
+   * Delete the pending requests with the id of the error
+   *
+   * @param genericMessage the message received
+   * @param subscribeRequests the pending subscribe requests
+   * @param catchupRequests the pending catchup requests
+   * @param createLaoRequests the pending create lao requests
+   */
+  public static void handleError(
```

Comment:
I infer from this that once you request the LAORepository to do something over the network, the LaoRepository RPC has no way of telling you if it failed. 

It might be interesting to implement a Promise-like pattern, where I can call the RPC and the LaoRepository makes sure that I get (asynchronous) feedback from the RPC, so that I can display (for instance) a spinner while the operation gets processed, or retry if there's an error.

I seem to remember that a LAORepository user can `subscribe` to an RPC to get the response, isn't a similar mechanism available for errors? If so, a `Promise`-like solution might be more appropriate.

Obviously, this is a completely separate issue from this PR :)